budgie.adoc

```diff
@@ -0,0 +1,36 @@
+=== Budgie desktop on FreeBSD
+
+Links: +
+link:https://blog.buddiesofbudgie.org/[Buddies of Budgie news] URL: link:https://blog.buddiesofbudgie.org/[https://blog.buddiesofbudgie.org/]
+link:https://codeberg.org/olivierd/freebsd-ports-budgie[Development repository] URL: link:https://codeberg.org/olivierd/freebsd-ports-budgie[https://codeberg.org/olivierd/freebsd-ports-budgie] +
+
+Contact: Olivier Duchateau <duchateau.olivier@gmail.com>
+
+Budgie initially developed as the default desktop environment for the former Evolve OS. Since the 10.6.x releases, improvements have been made to be "agnostic".
+
+It is build on top of the GNOME technologies such as GTK >= 3, GLib, Mutter, libpeas.
+
+The goal is to have a new desktop for end-users. I have submitted 2 reviews (link:https://reviews.freebsd.org/D37224[D37224] and link:https://reviews.freebsd.org/D37286[D37286] for Porter Handbook) in order committers import it.
+
+We can find:
```

Comment:
I'd reword that as "These reviews include:" as this sounds stilted to me, but it may be a matter of taste.